src/configure.ac

```diff
@@ -543,6 +543,9 @@ AC_CONFIG_FILES([
        Makefile
 ])
 
+# Pass into avrdude.conf.in
+AVRDUDE_FULL_VERSION='$(VERSION)'
```

Comment:
I'm no autotools expert (nor user for that matter), but my guess is that one more line is missing to make it work:
```autoconf
AC_SUBST(AVRDUDE_FULL_VERSION, $AVRDUDE_FULL_VERSION)
```

Reply:
maybe even:
```autoconf
AC_SUBST(AVRDUDE_FULL_VERSION, '$(VERSION)')
```
instead